app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RestaurantController < ApplicationController
```

Comment:
This class (and file name) should be called `RestaurantsController` and `restaurants_controller.rb` 
Same thing for cuisines.